<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="Image_Reverse_Dialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Reverse</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="folderLineEdit">
       <property name="toolTip">
        <string>The Folder That Will Be Scaned For A Valid Image Seq</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>Open File Explorer</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Run_Scan_Button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Exactly What The Button States
If Its Disabled Then The Input Folder Does Not Exist</string>
       </property>
       <property name="text">
        <string>Run Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sequence Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSpinBox" name="end_frame_input">
            <property name="toolTip">
             <string>Refer To Start Frame Tool Tip</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>End Frame : </string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSpinBox" name="frame_padding_input">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Refer To End Frame Tool Tip</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="prefix">
             <string>Frame Padding : </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="start_frame_input">
            <property name="toolTip">
             <string>Refer To Frame Count Tool Tip</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="prefix">
             <string>Start Frame : </string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="frame_count_input">
            <property name="toolTip">
             <string>If You Don't Know What This Means.
Then Just Stop What Your Doing.
Go Home.
Sit In A Cornner And Cry.</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>Frame Count: </string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="file_name_label">
            <property name="text">
             <string>File Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="file_name_input">
            <property name="toolTip">
             <string>The Name Of The File Sequecne Found</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="file_ext_label">
            <property name="text">
             <string>File Ext</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="file_ext_input">
            <property name="toolTip">
             <string>Lets See MMMMMM
What Do You Think :)</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="create_rev_seq_Button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>This Will Create A Folder Within The input Folder Called Revered
Containing The Reversed Images</string>
         </property>
         <property name="text">
          <string>Create Reverse Sequence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="account_for_looping">
         <property name="text">
          <string>Account For Looping</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="toolTip">
        <string>Unlike Life's Socal Standers.
This Will Show Progress</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Image_Reverse_Dialog</class>
   <extends>QDialog</extends>
   <header>Image_Reverse_Dialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>get_Image_Seq_Folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>on_folder_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Run_Scan_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>do_Folder_Scan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>create_rev_seq_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>do_Create_Rev_Image_Sequences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>on_folder_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>get_Image_Seq_Folder()</slot>
  <slot>do_Folder_Scan()</slot>
  <slot>do_Create_Rev_Image_Sequences()</slot>
  <slot>on_folder_changed()</slot>
 </slots>
</ui>
